### ISSUE TITLE
Add "Clear Browser Caches" to the top right menu in the Browser

### ISSUE THREAD

**Issue by tomekalphawallet** [AlphaWallet/alpha-wallet-android#1597]
![2020-09-11 13 19 13](https://user-images.githubusercontent.com/51817359/92924802-68f9d600-f431-11ea-8049-11987b62f44d.jpg)

**Comment by hboon** [comment]
A reason to prefer to keep it in Settings and not in the browser tab — it’s rarely used.

**Comment by tomekalphawallet** [comment]
Hmmm... How do you know that? It that an assumption or a proven by numbers fact?

**Comment by hboon** [comment]
Just an opinion to weigh.

I attribute that to intuition developed through 20 years of designing, building and selling software, many involving users and user interfaces, including working on macOS and iOS apps since before there’s an iPhoneOS SDK.

Also plenty of time spent studying user interface research and content.

Off the top of my head, the browsers I can think of all hide this functionality in some kind of settings menu.

But no, I can’t be absolutely sure. But if this is to suggest that we want to look at analytics to decide — I don’t have anything particular against analytics. Analytics is an indispensable tool, but with a small user base, analytics really isn’t that helpful and can be really misleading. There is much to do, and we ought to focus and prioritise on the most urgent matters.

Maybe you or others on the team are more experienced and know more about this. My point is not to compare years of experience (the absolute number is meaningless), but to point out that it’s not a blind guess.

And again, just a simple opinion on what I thought is obvious.

**Comment by tomekalphawallet** [comment]
Ok, let's close the issue. Don't want to even comment on this.

**Comment by tomekalphawallet** [comment]
Today, Aleks asked if we have an option to clear caches as wanted to check if the Browser page works fine even in our app. Then he used it a lot. Later, when working on the content, I cleared It out about 150 times.

When I asked my girlfriend to clear caches she could not find a way to do it. First, she hit the three dots. 

I will run 2 user tests tomorrow to see if users can find it or not. If our app is also targeted to developers, they will use this feature a lot. 

This is not urgent, it can wait. And you don’t have to do this on iOS. I requested for Android so I am not taking your bandwidth. 

Congrats with your experience. Good to have such experienced team mates! 👍
